Packs/NTT_Cyber_Threat_Sensor/Playbooks/Handle_TD_events_README.md

```diff
@@ -25,9 +26,9 @@ There are no inputs for this playbook.
 
 | **Path** | **Description** | **Type** |
 | --- | --- | --- |
-| CTS.EventID | Event ID | string |
-| CTS.OccuredTime | Event Time | unknown |
+| CTS.EventID | CTS EventID aka alert / sha | string |
+| CTS.OccuredTime | Timestamp when incident was registered | date |
 
 ## Playbook Image
 ---
-![Handle TD events](Insert the link to your image here)
+![Handle TD events](Insert the link to your image here)
```

Comment:
Please insert the playbook image here.
You can find the instructions here: https://xsoar.pan.dev/docs/integrations/integration-docs#images